apps/pipelines/graph.py

```diff
@@ -42,7 +42,8 @@ def build_runnable(self) -> RunnableSequence:
         state_graph = StateGraph(PipelineState)
         for node in self.nodes:
             node_class = getattr(nodes, node.type)
-            state_graph.add_node(node.id, node_class.get_callable(node))
+            node_instance = node_class(**node.params)
```

Comment:
Should we catch build errors here, to show somewhere as we used to in `.build`?

Reply:
https://github.com/dimagi/open-chat-studio/pull/474/commits/aa06189b084668fa07aefd0053475a4fef2718fe